src/main/scala/elo/package.scala

```diff
@@ -27,7 +27,11 @@ package object elo {
   // Utility functions
 
   // formats user score to be rounded to two decimal places
+  @deprecated(message = "use Player.formattedScore")
```

Comment:
Why decorate this with depracated instead of just removing it?

Reply:
This is still being used across a few places in Bot, removing all references would be better done in a separate PR.

Reply:
👍